cli/admin/src/test/java/io/pravega/cli/admin/serializers/SerializerTest.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.serializers;
+
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+import static io.pravega.cli.admin.utils.SerializerUtils.appendField;
+import static org.junit.Assert.assertEquals;
+
+public class SerializerTest {
+
+    @Test
+    public void testSltsChunkMetadataSerializer() {
+        StringBuilder userGeneratedMetadataBuilder = new StringBuilder();
+        appendField(userGeneratedMetadataBuilder, "key", "k");
```

Comment:
this is yet another example where having constants would help

Reply:
Constants are being used now.